runtime/experimental/compiler.Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM amazonlinux:2017.03 as compiler
+LABEL authors="Bubba Hines <bubba@stechstudio.com>"
+LABEL vendor1="Signature Tech Studio, Inc."
+LABEL vendor2="bref"
+LABEL home="https://github.com/mnapoli/bref"
+
+ENV DEPS="/deps" \
+    TARGET="/opt/bref"
+
+RUN mkdir -p ${DEPS}  \
+    ${TARGET}/bin \
+    ${TARGET}/doc \
+    ${TARGET}/etc \
+    ${TARGET}/include \
+    ${TARGET}/lib \
+    ${TARGET}/lib64 \
+    ${TARGET}/libexec \
+    ${TARGET}/sbin \
+    ${TARGET}/share
+
+# Working Directory
+WORKDIR ${TARGET}
+
+# Lambda is based on 2017.03. Lock YUM to that release version.
+RUN sed -i 's/releasever=latest/releaserver=2017.03/' /etc/yum.conf
+RUN yum makecache
+
+# Tools we need
+RUN yum groupinstall -y "Development Tools"  --setopt=group_package_types=mandatory,default
+RUN yum install -y  jq \
+                    gperf \
+                    expect \
+                    gtk-doc \
+                    texlive \
+                    python35 \
+                    docbook2X \
+                    findutils \
+                    python35-pip
+
+RUN yum clean all
+
+# Install Ninja and Meson
+RUN curl -Ls https://github.com/ninja-build/ninja/releases/download/v1.8.2/ninja-linux.zip >> /tmp/ninja.zip && \
+    cd /tmp && unzip /tmp/ninja.zip && \
+    cp /tmp/ninja /usr/local/bin && \
+    /usr/bin/pip-3.5 install meson
+
+# Install the rust toolchain
```

Comment:
Same here, I'm not sure why we need rust

Reply:
Commit https://github.com/mnapoli/bref/pull/132/commits/2a4e4a986ed30c728c4d0dd58593c50abb9f0298 addresses this.